
<html>
  <head>
    <style>
      a { font: 13px Arial;
      color: #933300;
      text-decoration: none; }

      a.header {font-weight: bold;}
      
      a:hover {text-decoration: underline;}

      body { background-color: white;
      font: 12px Arial;
      margin: 50px 50px 50px 50px; }

      h2 { font: bold 13px Arial;}

      h3 { font: bold 12px Arial;
      color: #003366;}

      td {font: 13px Arial;}

      ul {line-height:150%}

      p {line-height:150%}
    </style>

    <?php $rootdir="http://zwdzwd.io/InfiniumAnnotation/current/"; ?>

    <title>InfiniumMethylation BeadChips Annotation</title>
  </head>

  <table>
    <tr>
      <a href="#current">Download</a> | 
      <a href="#usage">Usage</a> | 
      <a href="#reference">Reference</a> |
      <a href="#contact">Contact</a>
    </tr>
  </table>

  <hr>
  <a class="header" name="download">Download</a>
  
  <br /><br />
 
  Release Jul-11-2016<br />
  <h2>GRanges R objects</h2>
  <p>require the <i>GenomicRanges</i> package from Bioconductor</p>
  <ul>
    <li>GRCh38/hg38 annotation with suggested overall masking
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.hg38.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.hg38.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.hg38.rda">HM27</a>)
    </li>

    <li>GRCh37/hg19 annotation with suggested overall masking
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.rda">HM27</a>)
    </li>

    <li>Population-specific masking GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.pop.hg38.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.pop.hg38.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.pop.hg38.rda">HM27</a>)
    </li>

    <li>Population-specific masking GRCh37/hg19
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.pop.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.pop.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.pop.rda">HM27</a>)
    </li>

    <li>Population-specific SNP information GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/hg38.commonsnp.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hg38.commonsnp.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hg38.commonsnp.rda">HM27</a>)
    </li>

    <li>Manufacturer's manifest
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.original.rda">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.original.rda">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.original.rda">HM27</a>)
    </li>

  </ul>
  
  <h2>tab-delimited format</h2>

  <ul>
    <li>GRCh38/hg38 annotation with suggested overall masking
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.hg38.tsv">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.hg38.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.hg38.tsv">HM27</a>)
    </li>
  
    <li>GRCh37/hg19 annotation with suggested overall masking
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.tsv">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.tsv">HM27</a>)
    </li>
    
    <li>Population-specific masking GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.pop.hg38.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.pop.hg38.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.pop.hg38.tsv">HM27</a>)
    </li>

    <li>Population-specific masking GRCh37/hg19
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.pop.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.pop.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.pop.tsv">HM27</a>)
    </li>

    <li>Population-specific SNP information GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/hg38.commonsnp.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hg38.commonsnp.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hg38.commonsnp.tsv">HM27</a>)
    </li>

    <li>Manufacturer's manifest
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.manifest.original.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.manifest.original.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.manifest.original.tsv">HM27</a>)
    </li>

    <li>Non-uniqueness of 3' nested subsequence
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.3sub.copynumber.tsv">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm450/hm450.3sub.copynumber.tsv">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/hm27/hm27.3sub.copynumber.tsv">HM27</a>)
      <br />- <smaller>Bisulfite-Genomic copy number of 3'-subsequence of 11-50 bases in length</smaller>
    </li>

  </ul>

  <h2>Functional annotations</h2>

  <ul>
    <li>Gene, promoter, CpG island
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.manifest.hg38.gencode.v22.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.manifest.hg38.gencode.v22.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.manifest.hg38.gencode.v22.tsv">hm27</a>)
      <br />- <smaller>gene annotation based on GENCODE v22</smaller>
      <br />- <smaller>CpG island definition based on UCSC genome browser</smaller>
    </li>

    <li>
      Roadmap Epigenomics Mapping Consortium
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.hg19.REMC.chromHMM.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.hg19.REMC.chromHMM.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.hg19.REMC.chromHMM.tsv">hm27</a>)
      <br />- <smaller>15 chromHMM states, 127 samples</smaller>
    </li>

    <li>
      ENCODE chromHMM states
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.hg19.ENCODE.chromHMM.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.hg19.ENCODE.chromHMM.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.hg19.ENCODE.chromHMM.tsv">hm27</a>)
      <br />- <smaller>15 chromHMM states, 9 samples</smaller>
    </li>

    <li>
      Imprinted differentially methylated loci
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.hg19.imprinting.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.hg19.imprinting.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.hg19.imprinting.tsv">hm27</a>)
      <br />- <smaller>based on Court et al. 2014 Genome Research</smaller>
    </li>

    <li>
      ENCODE Transcription Factor Binding Sites
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.hg19.ENCODE.TFBS.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.hg19.ENCODE.TFBS.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.hg19.ENCODE.TFBS.tsv">hm27</a>)
      <br />- <smaller>171 TFs and 91 samples from ENCODE project</smaller>
      <br />- <smaller>include distance to summit</smaller>
    </li>

    <li>
      CTCF binding sites
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.hg19.CTCF.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.hg19.CTCF.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.hg19.CTCF.tsv">hm27</a>)
      <br />- <smaller>based on Wang et al. 2012 Genome Research</smaller>
    </li>
    
  </ul>

  <h2>Cross-species mapping</h2>

  <!-- For Patient-derived Xenograft models in mice. -->
  
  <ul>
    <li>Mouse mm10/GRCm38 mapping
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.map.mm10.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.map.mm10.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.map.mm10.tsv">hm27</a>)
    </li>
    
    <li>Mouse 3'-subsequence copy number based on mm10/GRCm38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20160711/EPIC/EPIC.3sub.copynumber.mm10.tsv">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm450/hm450.3sub.copynumber.mm10.tsv">hm450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/20160711/hm27/hm27.3sub.copynumber.mm10.tsv">hm27</a>)
    </li>
  </ul>

  
  See <a href="http://zwdzwd.io/InfiniumAnnotation/">here</a> for previous releases.<br /><br />

  <hr>
  <a class="header" name="usage">Usage</a>

  <p>
    <b>seqnames</b>, <b>start</b> and <b>end</b> - the location of the target (1-based coordinates, 2 nucleotides for CpG probes, or 1 nucleotide for CpH and SNP probes). strand is left as "*" always. Some erroneous CpH probe coordinates mapping information in the manufacturer's manifest have been corrected. SNP probe coordinates are provided.
    <br />

    <b>addressA</b> and <b>addressB</b> - addresses of probe A and B on the chip designated by the original manifest.
    <br />

    <b>channel</b> - "Both" for type II probes and "Grn"/"Red" for type I probes.
    <br />

    <b>designType</b> - either "I" or "II".
    <br />

    <b>nextBase</b> - the actual extension base (on the probe strand) after bisulfite conversion ("A" or "C" or "T"). Unmapped probe has extension base labeled in the original manifest.
    <br />

    <b>nextBaseRef</b> - the extension base (on the hybridized DNA) before bisulfite conversion ("A", "C", "G" or "T"). Unmapped probe has "NA".
    <br />

    <b>probeType</b> - either "cg", "ch" or "rs".
    <br />

    <b>orientation</b> - either "up" or "down" specifying whether the probe is positioned upstream or downstream the target.
    <br />

    <b>probeCpGcnt</b> - the number of CpG in the probe.
    <br />

    <b>context35</b> - the number of CpG in the [-35bp, +35bp] window.
    <br />

    <b>probeStart</b> and <b>probeEnd</b> - the mapped start and end position of the probe, it is always 50bp long.
    <br />

    <b>ProbeSeqA</b> and <b>ProbeSeqB</b> - the probe sequence for allele A and B.
    <br />

    <b>chrmA</b>, <b>begA</b>, <b>flag.A</b>, <b>mapQ.A</b>, <b>cigarA</b> - the mapping info for probe A. mapQ=mapping quality score, 0-60, with 60 being the best
    <br />

    <b>chrmB</b>, <b>begB</b>, <b>flag.B</b>, <b>mapQ.B</b>, <b>cigarB</b> - the mapping info for probe B.
    <br />

    <b>posMatch</b> - whether the mapping matches the original manifest.
    <br />

  </p>
  <hr style="border-top: dotted 1px;">
  <p>
    
    <b>MASK.mapping</b> - whether the probe is masked for mapping reason. Probes retained should have high quality (>10 on 0-60 scale) consistent (with designed MAPINFO) mapping (for both in the case of type I) without INDELs.
    <br />

    <b>MASK.typeINextBaseSwitch</b> - whether the probe has a SNP in the extension base that causes a color channel switch from the official annotation (described as color-channel-switching, or CCS SNP in the reference). These probes should be processed differently than designed (by summing up both color channels instead of just the annotated color channel).
    <!-- This is implemented in SeSAMe, available on research.vai.org/tools/ -->
    <br />

    <b>MASK.rmsk15</b> - whether the 15bp 3'-subsequence of the probe overlap with repeat masker, this MASK is NOT recommended.
    <br />

    <b>MASK.sub25.copy</b>, <b>MASK.sub30.copy</b>, <b>MASK.sub35.copy</b> and <b>MASK.sub40.copy</b> - whether the 25bp, 30bp, 35bp and 40bp 3'-subsequence of the probe is non-unique.
    <br />

    <b>MASK.snp5.common</b> - whether 5bp 3'-subsequence (including extension for typeII) overlap with any of the common SNPs from dbSNP (global MAF can be under 1%).
    <br />

    <b>MASK.snp5.GMAF1p</b> - whether 5bp 3'-subsequence (including extension for typeII) overlap with any of the SNPs with global MAF >1%.
    <br />

    <b>MASK.extBase</b> - probes masked for inconsistent extension base based on mapping.
    <br />

    <b>MASK.general</b> - recommended general purpose masking merged from "MASK.sub30.copy", "MASK.mapping", "MASK.extBase", "MASK.typeINextBaseSwitch" and "MASK.snp5.GMAF1p".
    <br />
  </p>

  <hr>
  <a class="header" name="reference">Reference</a>
  <p>
    Zhou W, Laird PW and Shen H: Comprehensive characterization, annotation and innovative use of Infinium DNA Methylation BeadChip probes. <em>Nucleic Acids Research</em> 2016, in press
  </p>

  <hr>
  <a class="header" name="contact">Contact</a>
  
  <p>
    Questions regarding this annotation can be addressed to <a href="mailto:Wanding.Zhou@vai.org">Wanding.Zhou@vai.org<a>.
  </p>

</html>
