
<html>
  <head>
    <style>
      a { font: 13px Arial;
      color: #933300;
      text-decoration: none; }

      a.header {font-weight: bold;}
      
      a:hover {text-decoration: underline;}

      body { background-color: white;
      font: 12px Arial;
      margin: 50px 50px 50px 50px; }

      h2 { font: bold 13px Arial;}

      h3 { font: bold 12px Arial;
      color: #003366;}

      td {font: 13px Arial;}

      ul {line-height:150%}

      p {line-height:150%}
    </style>

    <?php $rootdir="http://zwdzwd.io/InfiniumAnnotation/current/"; ?>

    <title>InfiniumMethylation BeadChips Annotation</title>
  </head>

  <table>
    <tr>
      <a href="#current">Release Notes</a> | 
      <a href="#download">Lastest Release Download</a> | 
      <a href="#usage">Usage</a> | 
      <a href="#citation">Citation</a> |
      <a href="#contact">Contact</a> |
      <a href="#crossspecies">Cross-species</a>
    </tr>
  </table>

  <hr>
  <a class="header" name="release">Release Notes</a>

  <br /><br />
  <b>Update Jul-4-2020</b>  Updates: <br />
  <ul>
    <li>More detailed annotation to SNP and channel-switching probes</li>
  </ul>
  
  <b>Release Sep-9-2018 (latest)</b>  Updates: <br />
  <ul>
    <li>Added gene_HGNC column for gene symbols corrected for HGNC compatibility</li>
    <li>Updated SNP masking with dbSNP build 151</li>
    <li>All maskings were redone on hg19. Previously hg19 manifest file was "borrowing" hg38 maskings.</li>
    <li>CCS probes have shrunk in numbers based on AF filtering (>1%).</li>
    <li>See <a href="http://rpubs.com/zhouwanding/417953">Updates Summary</a> for more details.</li>
  </ul>

  <b>Release Aug-8-2018</b> Updates: <br />
  <ul>
    <li>Fix to MASK_mapping, some mapping with NM>0 are now masked as well </li>
    <li>Addition of NM columns and gene annotation to the manifest file.</li>
    <li>Previously, masking of mapping issues were merged from hg19 and hg38. Now masking on hg19 and hg38 was re-built entirely independently. There is some small decrease in the number of masking due to this change and a small increase due to the added NM-based masking (included in MASK_mapping).</li>
    <li>See <a href="http://rpubs.com/zhouwanding/InfiniumMethylation20180808">Updates Summary</a> for more details.</li>
  </ul>

  <p><b>Release Mar-4-2018</b> Fix to hg19 decoy mapping inconsistency. </p>

  <p><b>Release Jan-4-2018</b> Updated strand information. </p>
  
  <b>Release Nov-23-2017</b> Updates: <br />
  <ul>
    <li>Changed to RDS from RData following R's recommendation.</li>
    <li>Update to missing value representation.</li>
    <li>Update to table headers.</li>
    <li>Added gene annotation in short forms.</li>
    <li>Added mapping to genome both including and excluding alt-chromosomes.</li>
    <li>Made consistent naming of columns with underscores.</li>
  </ul>

  <b>Release Mar-13-2017</b> Updates: fix to relative position in SNP masking of type-I probes. <br />
  
  <hr>
  <a class="header" name="download">Latest Release Download</a>

  <br />
  <p><b>See <a href="http://zwdzwd.io/InfiniumAnnotation/">here</a> for previous releases.</b><br />
  <u>Note: some old gz files appear double compressed when downloaded with filefox. Please apply decompression twice.</u></p>

  <h2>RDS Files</h2>
  <p><u>require the <i>GenomicRanges</i> package from Bioconductor</u></p>
  <ul>
    <li>Basic <span style="font-weight:bold;color:DodgerBlue">hg38</span> annotation with suggested overall masking (<b>MASK_general</b> column) <br />
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.manifest.rds">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.manifest.rds">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.manifest.rds">HM27</a>)
    </li>

    <li>Basic <span style="font-weight:bold;color:DodgerBlue">hg19</span> annotation with suggested overall masking (<b>MASK_general</b> column) <br />
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.manifest.rds">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.manifest.rds">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.manifest.rds">HM27</a>)
    </li>

    <li>Detailed gene, promoter, CpG island (including transcript ID and distance to TSS) <br>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.manifest.gencode.v22.rds">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.manifest.gencode.v22.rds">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.manifest.gencode.v22.rds">HM27</a>)
      <br />- <smaller>gene annotation based on GENCODE v22</smaller>
      <br />- <smaller>CpG island definition based on UCSC genome browser</smaller>
    </li>

    <li>Converting SNP to variant format - rs probes
      <span style="font-weight:bold;color:DodgerBlue">hg38</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.snp_overlap_b151.rds">EPIC</a>)
    </li>
    
    <li>Converting SNP to variant format - rs probes
      <span style="font-weight:bold;color:DodgerBlue">hg19</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg19.snp_overlap_b151.rds">EPIC</a>)
    </li>

    <li>Converting SNP to variant format - channel-switching Infinium-I probes
      <span style="font-weight:bold;color:DodgerBlue">hg38</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.typeI_overlap_b151.rds">EPIC</a>)
    </li>
    
    <li>Converting SNP to variant format - channel-switching Infinium-I probes
      <span style="font-weight:bold;color:DodgerBlue">hg19</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg19.typeI_overlap_b151.rds">EPIC</a>)
    </li>
    
  </ul>
  
  <h2>Plain Text Files (tab-delimited)</h2>
  <ul>
    <li>Basic <span style="font-weight:bold;color:DodgerBlue">hg38</span> annotation with suggested overall masking (<b>MASK_general</b> column) <br>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.manifest.tsv.gz">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.manifest.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.manifest.tsv.gz">HM27</a>)
    </li>
  
    <li>Basic <span style="font-weight:bold;color:DodgerBlue">hg19</span> annotation with suggested overall masking (<b>MASK_general</b> column) <br>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.manifest.tsv.gz">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.manifest.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.manifest.tsv.gz">HM27</a>)
    </li>
    
    <li>Detailed gene, promoter, CpG island (including transcript ID and distance to TSS) <br>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.manifest.gencode.v22.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.manifest.gencode.v22.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.manifest.gencode.v22.tsv.gz">HM27</a>)
      <br />- <smaller>gene annotation based on GENCODE v22</smaller>
      <br />- <smaller>CpG island definition based on UCSC genome browser</smaller>
    </li>

    <li>Converting SNP to variant format - rs probes
      <span style="font-weight:bold;color:DodgerBlue">hg38</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.snp_overlap_b151.tsv.gz">EPIC</a>)
    </li>
    
    <li>Converting SNP to variant format - rs probes
      <span style="font-weight:bold;color:DodgerBlue">hg19</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg19.snp_overlap_b151.tsv.gz">EPIC</a>)
    </li>

    <li>Converting SNP to variant format - channel-switching Infinium-I probes
      <span style="font-weight:bold;color:DodgerBlue">hg38</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg38.typeI_overlap_b151.tsv.gz">EPIC</a>)
    </li>
    
    <li>Converting SNP to variant format - channel-switching Infinium-I probes
      <span style="font-weight:bold;color:DodgerBlue">hg19</span>
      (<a href="http://zwdzwd.io/InfiniumAnnotation/20200704/EPIC/EPIC.hg19.typeI_overlap_b151.tsv.gz">EPIC</a>)
    </li>
    
    
    <br>

    <li>Population-specific masking GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.manifest.pop.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.manifest.pop.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.manifest.pop.tsv.gz">HM27</a>)
    </li>

    <li>Population-specific masking GRCh37/hg19
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.manifest.pop.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.manifest.pop.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.manifest.pop.tsv.gz">HM27</a>)
    </li>

    <li>Population-specific SNP information GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.commonsnp.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.commonsnp.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.commonsnp.tsv.gz">HM27</a>)
    </li>

    <li>Manufacturer's manifest GRCh37/hg19
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.manifest.original.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.manifest.original.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.manifest.original.tsv.gz">HM27</a>)
    </li>

    <li>Non-uniqueness of 3' nested subsequence GRCh38/hg38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg38.3sub.copynumber.tsv.gz">EPIC</a>, 
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg38.3sub.copynumber.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg38.3sub.copynumber.tsv.gz">HM27</a>)
      <br />- <smaller>Bisulfite-Genomic copy number of 3'-subsequence of 11-50 bases in length</smaller>
    </li>

    <li>
      Roadmap Epigenomics Mapping Consortium
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.REMC.chromHMM.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.REMC.chromHMM.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.REMC.chromHMM.tsv.gz">HM27</a>)
      <br />- <smaller>15 chromHMM states, 127 samples</smaller>
    </li>

    <li>
      ENCODE chromHMM states
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.ENCODE.chromHMM.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.ENCODE.chromHMM.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.ENCODE.chromHMM.tsv.gz">HM27</a>)
      <br />- <smaller>15 chromHMM states, 9 samples</smaller>
    </li>

    <li>
      Imprinted differentially methylated loci
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.imprinting.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.imprinting.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.imprinting.tsv.gz">HM27</a>)
      <br />- <smaller>based on Court et al. 2014 Genome Research</smaller>
    </li>

    <li>
      ENCODE Transcription Factor Binding Sites
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.ENCODE.TFBS.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.ENCODE.TFBS.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.ENCODE.TFBS.tsv.gz">HM27</a>)
      <br />- <smaller>171 TFs and 91 samples from ENCODE project</smaller>
      <br />- <smaller>include distance to summit</smaller>
    </li>

    <li>
      CTCF binding sites
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.hg19.CTCF.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.hg19.CTCF.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.hg19.CTCF.tsv.gz">HM27</a>)
      <br />- <smaller>based on Wang et al. 2012 Genome Research</smaller>
    </li>
    
  </ul>

  <hr>
  <a class="header" name="usage">Column Legends</a>
  <p>
    <b>seqnames</b>, <b>start</b> and <b>end</b> - the location of the target (1-based coordinates, 2 nucleotides for CpG probes, or 1 nucleotide for CpH and SNP probes). strand is left as "*" always. Some erroneous CpH probe coordinates mapping information in the manufacturer's manifest have been corrected. SNP probe coordinates are provided. <br />

    <b>strand</b> - This is consistent with "orientation" column for strand information of the actual probe. '+' is for all the up-probes positioned in smaller coordinates and '-' for all the down-probes positioned in greater coordinates with respect to the target CpGs.<br />

    <b>address_A</b> and <b>address_B</b> - addresses of probe A and B on the chip designated by the original manifest. <br />

    <b>channel</b> - "Both" for type II probes and "Grn"/"Red" for type I probes. <br />

    <b>designType</b> - either "I" or "II". <br />

    <b>nextBase</b> - the actual extension base (on the probe strand) after bisulfite conversion ("A" or "C" or "T"). Unmapped probe has extension base labeled in the original manifest. <br />

    <b>nextBaseRef</b> - the extension base (on the hybridized/template DNA) before bisulfite conversion ("A", "C", "G" or "T"). Unmapped probe has "NA". <br />

    <b>probeType</b> - either "cg", "ch" or "rs". <br />

    <b>orientation</b> - either "up" or "down" specifying whether the probe is positioned upstream (in smaller coordinates) or downstream (in greater coordinates) the target. <br />
    <b style="color:DodgerBlue;">Note that by design, probes positioned upstream (in smaller coordinates) are always on the Watson strand and probes positioned downstream (in greater coordinates) are always on the Crick strand.</b><br />

    <b>probeCpGcnt</b> - the number of additional CpGs in the probe (not counting the interrogated CpG). <br />

    <b>context35</b> - the number of CpG in the [-35bp, +35bp] window. <br />

    <b>probeStart</b> and <b>probeEnd</b> - the mapped start and end position of the probe, it is always 50bp long. <br />

    <b>ProbeSeq_A</b> and <b>ProbeSeq_B</b> - the probe sequence for allele A and B.<br />

    <b>gene</b> - ";"-separated list of gene annotations (unique and alphabetically sorted). Gene models follows GENCODE version 22 (hg38). <br />

    <b>gene_HGNC</b> - ";"-separated list of gene annotations (unique and alphabetically sorted). Genes are checked using HGNChelper for compatibility with HGNC. Gene models follows GENCODE version 22 (hg38). <br />

    <b>chrm_A</b>, <b>beg_A</b>, <b>flag_A</b>, <b>mapQ_A</b>, <b>cigar_A</b>, <b>NM_A</b> - the mapping info for probe A excluding decoy chromsomes. mapQ=mapping quality score, 0-60, with 60 being the best.<br />

    <b>chrm_B</b>, <b>beg_B</b>, <b>flag_B</b>, <b>mapQ_B</b>, <b>cigar_B</b>, <b>NM_B</b> - the mapping info for probe B excluding decoy chromosomes, like above.<br />

    <b>wDecoy_chrm_A</b>, <b>wDecoy_beg_A</b>, <b>wDecoy_mapQ_A</b>, <b>wDecoy_cigar_A</b>, <b>wDecoy_NM_A</b> - the mapping info for probe A including decoy chromosomes.<br />

    <b>wDecoy_chrm_B</b>, <b>wDecoy_beg_B</b>, <b>wDecoy_mapQ_B</b>, <b>wDecoy_cigar_B</b>, <b>wDecoy_NM_B</b> - the mapping info for probe B including decoy chromosomes.<br />

    <b>posMatch</b> - whether the mapping matches the original manifest, it only applies to hg19 and will be NA under hg38. <br />

  </p>
  <hr style="border-top: dotted 1px;">
  <p>
    <b>MASK.mapping</b> - whether the probe is masked for mapping reason. Probes retained should have high quality (>=40 on 0-60 scale) consistent (with designed MAPINFO) mapping (for both in the case of type I) without INDELs.
    <br />

    <b>MASK.typeINextBaseSwitch</b> - whether the probe has a SNP in the extension base that causes a color channel switch from the official annotation (described as color-channel-switching, or CCS SNP in the reference). These probes should be processed differently than designed (by summing up both color channels instead of just the annotated color channel).
    <!-- This is implemented in SeSAMe, available on research.vai.org/tools/ -->
    <br />

    <b>MASK.rmsk15</b> - whether the 15bp 3'-subsequence of the probe overlap with repeat masker, this MASK is NOT recommended.
    <br />

    <b>MASK.sub25.copy</b>, <b>MASK.sub30.copy</b>, <b>MASK.sub35.copy</b> and <b>MASK.sub40.copy</b> - whether the 25bp, 30bp, 35bp and 40bp 3'-subsequence of the probe is non-unique.
    <br />

    <b>MASK.snp5.common</b> - whether 5bp 3'-subsequence (including extension for typeII) overlap with any of the common SNPs from dbSNP (global MAF can be under 1%).
    <br />

    <b>MASK.snp5.GMAF1p</b> - whether 5bp 3'-subsequence (including extension for typeII) overlap with any of the SNPs with global MAF >1%.
    <br />

    <b>MASK.extBase</b> - probes masked for extension base inconsistent with specified color channel (type-I) or CpG (type-II) based on mapping.
    <br />

    <b>MASK.general</b> - recommended general purpose masking merged from "MASK.sub30.copy", "MASK.mapping", "MASK.extBase", "MASK.typeINextBaseSwitch" and "MASK.snp5.GMAF1p".
    <br />
  </p>

  <hr>
  <a class="header" name="citation">Citation</a>
  <p>
    Zhou W, Laird PW and Shen H, <a href="https://academic.oup.com/nar/article/45/4/e22/2290930">Comprehensive characterization, annotation and innovative use of Infinium DNA Methylation BeadChip probes</a>, <b>Nucleic Acids Research</b> 2017
  </p>

  <hr>
  <a class="header" name="contact">Contact</a>
  <p>
    Questions regarding this annotation can be addressed to <a href="mailto:wanding.zhou@pennmedicine.upenn.edu">wanding.zhou@pennmedicine.upenn.edu<a>.
  </p>


  <hr>
  <a class="header" name="crossspecies">Cross-species mapping</a>
  <!-- For Patient-derived Xenograft models in mice. -->
  
  <ul>
    <li>Mouse mm10/GRCm38 mapping
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.mm10.map.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.mm10.map.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.mm10.map.tsv.gz">HM27</a>)
    </li>
    
    <li>Mouse 3'-subsequence copy number based on mm10/GRCm38
      (<a href="http://zwdzwd.io/InfiniumAnnotation/current/EPIC/EPIC.mm10.3sub.copynumber.tsv.gz">EPIC</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM450/HM450.mm10.3sub.copynumber.tsv.gz">HM450</a>,
      <a href="http://zwdzwd.io/InfiniumAnnotation/current/HM27/HM27.mm10.3sub.copynumber.tsv.gz">HM27</a>)
    </li>
  </ul>

  <!-- More cross-species mapping can be found at -->
  <!-- <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1870-y">Needhamsen et al. 2017 BMC Bioinformatics</a> <br /> -->

</html>
